cabal.ghc964.Unix.project

```diff
@@ -0,0 +1,61 @@
+packages: ./ghcup.cabal
```

Comment:
There's already https://github.com/haskell/ghcup-hs/pull/1132 and we probably need to merge/fix that first.

Reply:
This PR has been merged